model_security_adjust_oaw/models/supplier_stock.py

```diff
@@ -135,10 +135,12 @@ def _get_ordered_amount(self):
                     ('state', 'in', ('draft', 'sent')),
                     ('product_id', '=', ss.product_id.id)
                 ])
-            print sol
+
             i = 0
             for line in sol:
                 if line.order_id.supplier_id == self.env.user.partner_id:
-                    print "Random"
-                    i += 1
+                    if line.qty > 1:
```

Comment:
I think this part can be simply like `i += line.product_uom_qty`, since qty is not a field in the `sale.order.line` and I think we do not need to consider case that the line is with 0 quantity